xoinvader/tests/chunk_duplicate_names.bg

```diff
@@ -0,0 +1,5 @@
+~chunk~ same
```

Comment:
oh, for such files create please directory in tests called 'fixtures', AFAIK this is for such things =)